sdk/PrebidMobile/PBTargetingParams.m

```diff
@@ -88,6 +95,102 @@ - (void)removeCustomKeywordWithKey:(NSString *)key {
     }
 }
 
+#pragma mark User Keywords
+
+- (void)setUserKeywords:(nonnull NSString *)key
+                 withValue:(nonnull NSString *)value {
+    if (_userKeywords == nil) {
+        return;
```

Comment:
Should here be initiating _userKeywords if nil?

Reply:
initialization is already done and its a shared instance so no initialization needed again